backend/pluto/pluto/_internal/server/flask_server.py

```diff
@@ -74,11 +90,68 @@ def run(self, **kwargs):
     def add_expense(self):
         expense_dict = request.get_json(force=True)
         expense_service = ExpenseServiceImpl(self.db)
-        expense_service.add_expense(expense_dict)
+        expense_service.add_expense_from_dict_without_id(expense_dict)
         return f"Inseriu {expense_dict} com sucesso!"
 
     def add_income(self):
         income_dict = request.get_json(force=True)
         income_service = IncomeServiceImpl(self.db)
         income_service.add_income(income_dict)
         return f"Inseriu {income_dict} com sucesso!"
+
+    # Based on: https://flask.palletsprojects.com/en/2.3.x/patterns/fileuploads/
+    def expense_file_upload(self):
+        base_html_return = """
+        <!doctype html>
+        <title>Upload new File</title>
+        <h1>Upload new File</h1>
+        <form action="/upload/expenses/" method=post enctype=multipart/form-data>
+        <input type="hidden" id="user_id" name="user_id" value="3487">
+        <input type=file name=file>
+        <input type=submit value=Upload>
+        </form>
+        """
```

Comment:
Will this be what the user sees on the UI? I believe this should be implemented on the front end, no?

Reply:
Isso não vai ser o que o usuário verá, foi só pra eu testar. Posso retirar para fazer o merge